### PR TITLE
Add diagrammatic `humility net counters` output

### PR DESCRIPTION
**Pull request by mkeeter**
Screenshot 2024-05-24 at 3 06 54 PM

### CODE REVIEW

File: cmd/net/src/lib.rs [hunk 1]
```diff
@@ -57,7 +57,17 @@ enum NetCommand {
...
     /// Print the counters
-    Counters,
+    ///
+    /// This is a destructive operation that clears counter values, resetting
+    /// them to zero.
+    Counters {
+        /// Show a table of raw counter values
+        #[clap(short, long)]
+        table: bool,
+        /// Show a diagram of good packet counts in context
```
Comment (hawkw) [inline cmd/net/src/lib.rs:67]: very nitpicky: do you think the help text should state that both are displayed if neither flags are present?

File: cmd/net/src/lib.rs [hunk 2]
```diff
@@ -473,6 +503,89 @@ fn net_counters(context: &mut ExecutionContext) -> Result<()> {
...
+        ksz_rx[2].to_string().green(),
+        ksz_tx[2].to_string().green(),
+        ksz_tx[1].to_string().green(),
+        v_media_rx[1].to_string().green(),
+        mac(v_mac_tx[1]),
+        ksz_rx[1].to_string().green(),
+        v_media_tx[1].to_string().green(),
+        mac(v_mac_rx[1]),
+        "MEDIA".dimmed(),
+        "MAC".dimmed()
```
Comment (hawkw) [inline cmd/net/src/lib.rs:584]: nit, take it or leave it: this might be a lot easier to figure out if we used named parameters in the format placeholders? OTTOH the Unicode art would be less well aligned, so I dunno...

**Review by hawkw — APPROVED** [review]
overall, this looks good to me! it's a little bit hard to parse the format string that relies on positional ordering, so i might prefer to use named params there, but, take it or leave it.

File: cmd/net/src/lib.rs (hunk 1 above)
Comment (mkeeter) [reply cmd/net/src/lib.rs:69]: Sure, done in e89180467001c17d22745957982a6fb3068f0fb4

(also, TIL that `-h` and `--help` are different in `clap`)
